.tool-versions

```diff
@@ -0,0 +1 @@
+flutter 3.13.7-stable
```

Comment:
Is there a difference between `3.13.7-stable` and `3.13.7`?

Reply:
As discussed during the standup: it's asdf's way of dealing with [flutter channels](https://docs.flutter.dev/release/upgrade#switching-flutter-channels).
```
$ asdf list all flutter | tail
3.13.4-stable
3.15.0-15.1.pre-beta
3.13.5-stable
3.15.0-15.2.pre-beta
3.13.6-stable
3.16.0-0.1.pre-beta
3.13.7-stable
3.16.0-0.2.pre-beta
3.13.8-stable
3.16.0-0.3.pre-beta
```